File: solutions/beecrowd/2779/2779.py
stickers_total_count = int(input())
stickers_bought_count = int(input())

stickers_bought_unique = set()
for _ in range(stickers_bought_count):
    stickers_bought_unique.add(int(input()))

print(stickers_total_count - len(stickers_bought_unique))

File: solutions/beecrowd/2779/2779.cpp
#include <cstdint>
#include <iostream>
#include <set>
#include <string>

int main() {
    int16_t stickers_total_count, stickers_bought_count, sticker;
    std::set< int16_t > stickers_bought_unique;

    std::cin >> stickers_total_count;
    std::cin >> stickers_bought_count;

    while (std::cin >> sticker) {
        stickers_bought_unique.insert(sticker);
    }

    std::cout << stickers_total_count - stickers_bought_unique.size()
              << std::endl;

    return 0;
}
